sdk/samples/iot/paho_iot_hub_c2d_sample.c

```diff
@@ -82,24 +82,12 @@ static void create_and_configure_mqtt_client(void)
   int rc;
 
   // Reads in environment variables set by user for purposes of running sample.
-  rc = iot_sample_read_environment_variables(SAMPLE_TYPE, SAMPLE_NAME, &env_vars);
-  if (az_result_failed(rc))
-  {
-    IOT_SAMPLE_LOG_ERROR(
-        "Failed to read configuration from environment variables: az_result return code 0x%08x.",
-        rc);
-    exit(rc);
-  }
+  iot_sample_read_environment_variables(SAMPLE_TYPE, SAMPLE_NAME, &env_vars);
 
   // Build an MQTT endpoint c-string.
   char mqtt_endpoint_buffer[128];
-  rc = iot_sample_create_mqtt_endpoint(
+  iot_sample_create_mqtt_endpoint(
```

Comment:
>iot_sample_create_mqtt_endpoint [](start = 2, length = 31)

Are the changes relevant to PNP?

Reply:
No there are some files changed here that shouldn't be. As mentioned in another comment, there is some weird stuff going on with the diff when I rebase on master and have a PR not to master but a feature branch. Trying to figure out why.

Reply:
updated so diff is now fixed